projectDocs/dev/codingStandards.md

```diff
@@ -159,6 +159,69 @@ Type hints make reasoning about code much easier, and allow static analysis tool
 * Prefer union shorthand (`X | Y`) over explicitly using `typing.Union`.
   * Corollary: prefer `T | None` over `typing.Optional[T]`.
 
+### Calling non-python code
+
+Todo: Merge with section in #18207
```

Comment:
can you open a (draft) PR to prepare this?

Reply:
What do you mean? A draft pr with what against what?
I can also rebase this onto `wip-buildX64`, that means the section is automatically there.

Reply:
I figured this was significant, otherwise if its just handling merge conflicts, we can delete this comment.

Reply:
I'm not intending to change anything in the Calling non-python code section in #18207. I'm only intending to introduce a sub heading beneath it to explain how the ctypesHelper kan be used.
I can probably copy the section from #18207 integrally, then merge conflict handling should do the rest.